client/src/main/com/sinch/sdk/domains/verification/adapters/converters/VerificationsDtoConverter.java

```diff
@@ -279,210 +281,65 @@ public static VerificationReportRequestResourceDto convert(
     return dto;
   }
 
-  private static FlashcallVerificationReportRequestDto convert(
+  private static VerificationReportRequestParametersFlashCall convert(
       VerificationReportFlashCallRequestParameters client) {
 
-    FlashcallVerificationReportRequestFlashCallDto subfield =
-        new FlashcallVerificationReportRequestFlashCallDto();
-    client.getCli().ifPresent(subfield::cli);
+    VerificationReportRequestParametersFlashCallOptions.Builder subfield =
```

Comment:
Regarding the naming of the variable, `subfield` is not explicit enough. As we are currently working on this API, we know well that a request object contains some common properties + some options specific to the verification method. But if there is another complex object to add at the root of the request parameters, it can't be named "subfield" again. Hence I would suggest to rename it "options" of "flashCallOptions" to better describe what it is.

Reply:
Yes.
Already pointed here (I did not forget 😄 ): https://github.com/sinch/sinch-sdk-java/pull/81#discussion_r1615753291
This can be done with this PR; no longer conflicts to be managed across stack of pending PRs 
Done: